src/components/MapBox.tsx

```diff
@@ -117,12 +117,9 @@ export const MapBox: React.FC = () => {
   const { selectedFacilityId } = useAppState();
   const dispatch = useAppDispatch();
 
-  // to highlight a given marker use url params "focusedFacilityId"
+  // to highlight a given event marker use url params "focusedEvent"
   const [searchParams] = useSearchParams();
-  const focusedFacilityId = useMemo(
-    () => searchParams.get('focusedFacilityId'),
-    [searchParams]
-  );
+  const focusedEvent = useMemo(() => searchParams.get('focusedEvent'), [searchParams]);
 
```

Comment:
focusedFacilityId was made for centering accommodation on map after see map button on acccommodation page. Is it not required anymore or handled in other place?

Reply:
Yea, it is no longer needed because `See map` no longer returns to the `Search Page` but currently opens google maps in a new tab